release-notes/opensearch-index-management-dashboards-plugin.release-notes-1.3.6.0.md

```diff
@@ -0,0 +1,9 @@
+## Version 1.3.6.0 2022-10-06
+Compatible with OpenSearchDashboards 1.3.0
```

Comment:
I think we may need to adjust the version on line 2 to `1.3.6`. In the following examples, we adjust the `compatible with...` line to specify the incremental version as well.
https://github.com/opensearch-project/dashboards-reports/blob/1.3/release-notes/opensearch-dashboards-reports.release-notes-1.3.6.0.md
https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/release-notes/opensearch-alerting-dashboards-plugin.release-notes-2.0.1.0.md?plain=1#L2

Reply:
got it